wagn/features/pointer_inputs.feature

```diff
@@ -38,7 +38,7 @@ Feature: Pointer Inputs
     Given I create Phrase card "User+*type+*structure" with content "{{+friends}}"
     And I create Phrase card "friends+*right+*input" with content "multiselect"
     When I edit "Joe User"
-    And I select "Joe Admin" from "pointer_multiselect"
+    And I select "Joe Admin" from "friends"
```

Comment:
The correct label is with "+". I forgot the "+" here but it doesn't matter because I remove the "+" in the step definition anyway. I'm not able to find the label with the "+" because it is in a separate span tag.